g2e.py

```diff
@@ -0,0 +1,84 @@
+"""Uploads stuff from GitHub to Edux"""
+import io
+import os
+import pathlib
+import re
+import subprocess
+
```

Comment:
Delete this line

Reply:
why?

Reply:
(it splits standard library modules from pip installed ones)

Reply:
Well, why is it there in first place? I just thought, why not to make it nicer... :)

Reply:
Aha, okay make sense... 👍